MatrixSDK/JSONModels/MXEvent.m

```diff
@@ -196,20 +196,30 @@ - (NSUInteger)age
     return age;
 }
 
-- (NSDictionary *)originalDictionary
+- (NSDictionary *)JSONDictionary
```

Comment:
Agreed. The issue is that the hs fills the event differently according if it comes from /sync v2 or from /messages.